docs/4.4/api-reference/directives.md

```diff
@@ -2541,6 +2541,64 @@ type Mutation {
 }
 ```
 
+## @upsert
```

Comment:
We do not put new features in minor releases. The reason for that is that somebody on `4.4.0` might read something in the docs that does not apply to their version.